views/js/qtiCreator/helper/itemLoader.js

```diff
@@ -55,8 +55,13 @@ define([
                 // request doesn't handle empty response with 200 code. See: core/request.js:240
                 const itemRdf = request(config.itemDataUrl, { uri: config.uri }).catch(d => d);
 
+                const containsOnlyIdentifier = (data) => {
```

Comment:
**question:** is Prettier properly configured? lone parameters should not have parenthesis...